docs/usage/getting-started/private-packages.md

```diff
@@ -265,6 +265,34 @@ Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.y
 Renovate updates `npmRegistries` in `.yarnrc.yml` with resolved `hostRules` before running Yarn.
 For Renovate to overwrite existing `npmRegistries` entry, the key should match the `matchHost` without the protocol (`http:` or `https:`) and with the trailing slash.
 
+For example, the Renovate configuration:
+
+```json
+{
+  "hostRules": [
+    {
+      "matchHost": "https://npm.pkg.github.com/",
+      "hostType": "npm",
+      "encrypted": {
+        "token": "<Encrypted PAT Token>"
+      }
+    }
+  ]
+}
+```
+
+will update `.yarnrc.yml` as following:
+
+```yaml
+npmRegistries:
+  //npm.pkg.github.com/:
+    npmAuthToken: <Decrypted PAT Token>
+  //npm.pkg.github.com:
+    # this will not be overwritten and may conflict
+  https://npm.pkg.github.com/:
+    # this will not be overwritten and may conflict
```

Comment:
My intention was that they would not be overwritten, but as they are valid Yarn configuration, it'd conflict with the configuration set by Renovate, so the the last sentence was to prevent it.

Reply:
Should we change our behavior or do you think it's ok?

Reply:
I don't have a strong opinion. The behavior can be changed here, though: https://github.com/renovatebot/renovate/blob/62a540c272d4f0b2738781d0442d61fd946cb482/lib/manager/npm/post-update/index.ts#L519-L522

Reply:
Let's merge this docs update though as it's useful already